docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -0,0 +1,76 @@
+<!--[metadata]>
++++
+title = "Create a Swarm"
+description = "Initialize the Swarm"
+keywords = ["tutorial, cluster management, swarm"]
+[menu.main]
+identifier="initialize-swarm"
+parent="swarm-tutorial"
+weight=12
++++
+<![end-metadata]-->
+
+## Create a Swarm
+
+After you install Docker on your networked machines and start the Docker Engine
```

Comment:
@sanscontext : Maybe so that we don't have the prereqs in a 3rd list , we can say "After you finish the tutorial setup?"